baseapp/baseapp.go

```diff
@@ -144,6 +144,9 @@ type BaseApp struct { //nolint: maligned
 	abciListeners []ABCIListener
 
 	chainID string
+
+	// StreamEvents
+	StreamEvents chan StreamEvents
```

Comment:
Where is this channel drained (read from)?

Reply:
It's read from injective-core > event stream